src/main/java/com/ratifire/devrate/exception/FeedbackFrequencyLimitReached.java

```diff
@@ -0,0 +1,15 @@
+package com.ratifire.devrate.exception;
+
+
+/**
+ * Exception thrown when a user reaches the feedback frequency limit.
+ */
+public class FeedbackFrequencyLimitReached extends RuntimeException {
```

Comment:
It whould be nice if it were called  FeedbackFrequency FeedbackSubmissionLimitException